tests/test_hermes_provider_parity.py

```
@@ -319,3 +322,44 @@ def test_provider_persona_tool_dispatch_matches(tmp_path, provider_modules):
+    sys.path.insert(0, str(PROJECT_ROOT))
+    try:
+        from mnemosyne.core.beam import BeamMemory
+
+        observed = {}
+        for name, module in provider_modules.items():
+            db_path = tmp_path / f"{name}-batch.db"
+            beam = BeamMemory(session_id=f"batch-{name}", db_path=str(db_path))
+            provider = module.MnemosyneMemoryProvider.__new__(module.MnemosyneMemoryProvider)
+            provider._beam = beam
+            provider._default_scope = "session"
+            provider._audit_event = lambda *args, **kwargs: None
```

Ruff (0.15.20)

[warning] Unused lambda argument: `args`

(ARG005)